src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1233,6 +1452,184 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+"""
+Settings that control the behavior of itinerary filtering. These settings are meant mainly
+for debugging purposes.
+"""
+input PlanItineraryFilterSettingsInput {
+    """
+    Itinerary filter debug profile used to control the behaviour of itinerary filters.
+    """
+    itineraryFilterDebugProfile: ItineraryFilterDebugProfile = OFF
+
+    """
+    Pick one itinerary from each group after putting itineraries that are `85%` similar together,
+    if the given value is `0.85`, for example. Itineraries are grouped together based on relative
+    the distance of travel that is identical between the itineraries.
+    """
+    groupSimilarityKeepOne: Float = 0.85
+
+    """
+    Pick three itineraries from each group after putting itineraries that are `68%` similar together,
+    if the given value is `0.68`, for example. The itineraries are grouped by similar legs
+    (on board same journey). The value must be at least `0.5`.
+    """
+    groupSimilarityKeepThree: Float = 0.68
```

Comment:
Should we use the WeightingFactor scalar for these, create a new one or just use Float? I copied the comments and slightly modified them from the transmodel API but there are some aspects I don't understand:
1. Are they really grouped together based on the distance of travel they share, not duration? Do all legs count or only transit legs?
2. Does `groupSimilarityKeepOne` also have a lower limit?